packages/Card/Card.jsx

```diff
@@ -71,9 +74,37 @@ const Card = ({ variant, children, fullHeight, spacing, ...rest }) => {
     spacingProps.inset = 4
   }
 
+  const ContainerStyles = imageProps =>
+    imageProps &&
+    imageProps.position &&
+    handleResponsiveStyles({ position: imageProps.position }, ({ position }) => {
+      if (!imageProps) return {}
+      const direction = {
+        left: 'row',
+        right: 'row-reverse',
+        top: 'column',
+        bottom: 'column-reverse',
+        none: 'row',
+      }
+      const styles = {
+        display: 'flex',
+        flexDirection: direction[position],
+        justifyContent: 'space-between',
+        '> img': {
+          display: position === 'none' ? 'none' : 'block',
+          margin: 'auto',
+        },
+      }
+      return styles
+    })
+
+  const StyledDiv = styled.div(ContainerStyles)
```

Comment:
`styled-components` should be declared outside of any render method. In this case the `StyledDiv` should be declared outside of `Card`, just as `StyledCard` is.

See https://styled-components.com/docs/faqs#why-should-i-avoid-declaring-styled-components-in-the-render-method

Reply:
Makes sense, thanks for the reference! I've fixed this.